presto-main/src/main/java/com/facebook/presto/metadata/SqlAggregationFunction.java

```diff
@@ -96,41 +106,27 @@ public boolean isDeterministic()
 
     public abstract InternalAggregationFunction specialize(BoundVariables boundVariables, int arity, TypeManager typeManager, FunctionRegistry functionRegistry);
 
-    public static class SimpleSqlAggregationFunction
+    public static class AnnotationBasedSqlAggregationFunction
```

Comment:
Maybe I don't have deep context, but to me it is not `AnnotationBased`. The only field seems to have nothing with annotations (or it has wrong name).